vnext/Desktop.IntegrationTests/DesktopTestRunner.cpp

```diff
@@ -20,7 +20,6 @@ using namespace facebook::xplat::module;
 using std::make_shared;
 using std::make_tuple;
 using std::make_unique;
-using std::move;
```

Comment:
Should we get rid of all the std::* using declarations?

Reply:
There has been a mostly stylistic discussion around this.
I currently don't agree with not using any `std`-prefixed types, but I can update functions to prevent an obscure but still possible arg lookup bug.